providers/os/connection/snapshot/blockdevices.go

```diff
@@ -33,7 +34,7 @@ type PartitionInfo struct {
 }
 
 func (cmdRunner *LocalCommandRunner) GetBlockDevices() (*BlockDevices, error) {
-	cmd, err := cmdRunner.RunCommand("lsblk -f --json")
+	cmd, err := cmdRunner.RunCommand("lsblk -bo NAME,SIZE,FSTYPE,MOUNTPOINT,LABEL,UUID --json")
```

Comment:
Just for my understanding, why are we adding those additional arguments? Are those fields are not covered with the default output?

Reply:
Nope, if you do `lsblk -f --json` the size isnt there. This also lets us extend this in the future. For example, some partitions have a `PARTLABEL` which we can use (e.g. in Azure some partitions can be labeled with the BOOT PARTLABEL so we can omit those)

Reply:
This just makes it easier for us in the future since all we neeed is a an extra column and an extra field in the struct.